src/mbgl/text/collision_tile.cpp

```diff
@@ -153,12 +155,27 @@ Box CollisionTile::getTreeBox(const Point<float>& anchor, const CollisionBox& bo
     };
 }
 
-std::vector<IndexedSubfeature> CollisionTile::queryRenderedSymbols(const mapbox::geometry::box<int16_t>& box, const float scale) {
+std::vector<IndexedSubfeature> CollisionTile::queryRenderedSymbols(const GeometryCollection& queryGeometry, const float scale) {
```

Comment:
I think we can simplify the parameter here, and various other places, to `const GeometryCoordinates& queryGeometry`. There's no need to support multipolygon query geometries: https://github.com/mapbox/mapbox-gl-native/commit/d7a816db0cdea3f2afc0ef234f71abfc721ad56b.